src/components/App/App.vue

```diff
@@ -43,9 +49,46 @@ export default {
       })
     },
     retrieveLocale (location) {
-      fetchAPI.getLocale(location.city, location.state).then(data => {
-        this.locations.push(data.data)
+      fetchAPI.getLocale(location.city, location.state).then(value => {
+        value.data.id = Date.now()
+        this.checkExistingLocations(value.data)
+        this.updateLocalStorage()
+      })
+    },
+    retrieveCurrent () {
```

Comment:
Let's make a card in Trello for adding a key/value to this so we can assign a class to the "Current Location" card and differentiate the styling/HTML for it.